OrbitGl/Track.cpp

```diff
@@ -10,7 +10,7 @@
 #include "absl/strings/str_format.h"
 
 float TEXT_Z = -0.004f;
-float TRACK_Z = -0.005f;
+float TRACK_Z = -0.1f;
```

Comment:
constexpr const float?

Reply:
It's handy to be able to tweak those at runtime.  I moved them to TimeGraphLayout and exposed them through the ImGui debug UI.